packages/api/src/SubmittableExtrinsic.ts

```diff
@@ -181,13 +184,13 @@ export default function createSubmittableExtrinsic<ApiType> (
         }
       },
       signAndSend: {
-        value: function (account: IKeyringPair | string | AccountId | Address, _options?: Partial<Partial<SignatureOptions>> | Callback<ISubmittableResult>, statusCb?: Callback<ISubmittableResult>): SumbitableResultResult<ApiType> | SumbitableResultSubscription<ApiType> {
+        value: function (account: IKeyringPair | string | AccountId | Address, optionsOrStatus?: Partial<Partial<SignatureOptions>> | Callback<ISubmittableResult>, statusCb?: Callback<ISubmittableResult>): SumbitableResultResult<ApiType> | SumbitableResultSubscription<ApiType> {
```

Comment:
unless I'm missing something, `Partial<Partial<T>> === Partial<T>`

Reply:
I dunno why it is like that, copied from the line above. We were both blind letting that slip in. Me now ... twice...